<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewBids</class>
 <widget class="QDialog" name="ViewBids">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>261</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Sell Bids</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Buy Bids</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Ok">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QListWidget" name="BidsListWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>460</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel Bid</string>
   </property>
  </widget>
  <widget class="QLabel" name="bid_status_label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Update">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>460</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Create">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>460</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
